src/recipe.cpp

```diff
@@ -729,32 +738,40 @@ std::set<proficiency_id> recipe::assist_proficiencies() const
     return ret;
 }
 
-float recipe::proficiency_time_maluses( const Character &guy ) const
+float recipe::proficiency_time_maluses( const Character &c ) const
```

Comment:
Can we keep the non-single variable name?

Reply:
Sure, but can you suggest something else that's short and not gender-specific?
"Character *c" is used in 6 other functions in this file, so I was going for consistency. I could change it to "crafter" instead (as is the norm in crafting.cpp).

Reply:
I don't really think it's worth the churn to change it. That said, that's because it hurts history, and you're already killing all the history with your functional changes. So, let's go with crafter.